Buckify MPS AOT components.

shoumikhin opened the pull request:
Summary: .

Differential Revision: D58394879

cccclai reviewed (approved):
Looks right to me. These tests are probably broken since the changes in torch.export earlier this year.

shoumikhin committed:
Buckify MPS AOT components. (#3925)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/3925

.

Reviewed By: cccclai

Differential Revision: D58394879
